src/components/actions/getApplicationData.ts

```diff
@@ -71,6 +71,9 @@ export const getApplicationData = async (input: {
       }
     : {}
 
+  const env = Object.fromEntries(
+    Object.entries(process.env).filter(([key, _]) => config.envVars?.includes(key))
+  )
```

Comment:
Expose environment variables, but only if defined in system preferences.

Reply:
Should it be exposeEnvVarsToApplicationData ? And mentioned in config.ts ?